tests/python/common/test_heterograph.py

```diff
@@ -1255,6 +1255,57 @@ def test_pin_memory_(idtype):
     assert not g3.is_pinned()
 
 
+@unittest.skipIf(
```

Comment:
Maybe create a test_heterograph_index.py and put the test case in it? In the future, we can gradually move all test related to heterograph_index.py to it.